### PR TITLE
py-gwpy: update to 0.11.0

### PR DESCRIPTION
**Pull request by duncanmmacleod**
Update `py-gwpy` port to 0.11.0.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.5 17F77
Xcode 9.4.1 9F2000

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by pmetzger** [comment]
LGTM.

File: python/py-gwpy/Portfile [hunk 1]
```diff
@@ -22,45 +22,44 @@ homepage            https://gwpy.github.io
...
-                        port:glue
+                        port:py${python.version}-ligo-segments \
+                        port:py${python.version}-tqdm
+
     if {${python.version} < 34} {
         depends_lib-append port:py${python.version}-enum34
     }
 
-    depends_build-append port:py${python.version}-setuptools
+    if {${python.version} == 27} {
```
Comment (pmetzger) [inline python/py-gwpy/Portfile]: You are only making the variants available under 2.7?

Comment (duncanmmacleod) [reply python/py-gwpy/Portfile]: @pmetzger, the extra dependencies included in the variants are only available (currently) for 2.7. If they were available more widely, I would include the variants for 3.6

Comment (pmetzger) [reply python/py-gwpy/Portfile]: Gotcha.

**Comment by pmetzger** [comment]
Merged. Thank you, @duncanmmacleod!

**Pull request merged**